### PR TITLE
Update contact form success banner

### PR DESCRIPTION
**Pull request by britneywwc**
## Done
- Format files with djlint (mainly what's causing the large diff)
- Change form submission redirect from `#success` to `#contact-form-success`
- Shows the correct message: **Thank you for contacting our team. We will be in touch shortly.** as seen in the forms copy doc

## QA
- Go to demos: [/solutions/ai](https://canonical-com-1267.demos.haus/solutions/ai), [/solutions/secure-open-source](https://canonical-com-1267.demos.haus/solutions/secure-open-source), [/solutions/big-data-ai](https://canonical-com-1267.demos.haus/solutions/big-data-ai)
- Click on `Contact Canonical` and submit form
- On form submission success, check that url has `#contact-form-success` and is showing the correct notification banner (in screenshot below)
 - Forms copy docs: [secure-open-source](https://docs.google.com/document/d/10SsItwbzMqZB4NzKANiOW9kx05j5QXH7wUoD269jebY/edit), [ai](https://docs.google.com/document/d/1rlAO5gLtxsmi3la_yk2rCBOTXBAph2pkRhwVyu6Ua-o/edit)
 - Form copy docs for [big-data-ai](https://docs.google.com/document/d/1ZDYZHdATF7Pqlmkq9s-ucN9YdCu7ZA648vlGkyQFApc/edit) does not specify a confirmation banner message, but changed for consistency

## Issue / Card

Fixes [WD-11848](https://warthogs.atlassian.net/browse/WD-11848)

## Screenshots
Screenshot 2024-06-05 at 2 25 23 PM

[WD-11848]: https://warthogs.atlassian.net/browse/WD-11848?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/solutions/ai/index.html [hunk 1]
```diff
@@ -637,7 +637,7 @@ <h2>
   <script defer src="{{ versioned_static('js/modals.js') }}"></script>
 
   {% with %}
-    {% set returnURL = "https://canonical.com/solutions/big-data-ai#success" %}
+    {% set returnURL = "https://canonical.com/solutions/ai#contact-form-success" %}
```
Comment (britneywwc) [inline templates/solutions/ai/index.html:640]: Change here

File: templates/solutions/big-data-ai/index.html [hunk 2]
```diff
@@ -1,432 +1,487 @@
...
 
-<script defer src="{{ versioned_static('js/modals.js') }}"></script>
+    <script defer src="{{ versioned_static('js/modals.js') }}"></script>
 
-{% with %}
-	{% set returnURL = "https://canonical.com/solutions/big-data-ai#success" %}
-	{% include "solutions/big-data-ai/_form-big-data-ai.html" %}
-{% endwith %}
+    {% with %}
+      {% set returnURL = "https://canonical.com/solutions/big-data-ai#contact-form-success" %}
```
Comment (britneywwc) [inline templates/solutions/big-data-ai/index.html:482]: Change here

File: templates/solutions/secure-open-source/index.html [hunk 3]
```diff
@@ -1,788 +1,850 @@
...
 
-<script defer src="{{ versioned_static('js/modals.js') }}"></script>
+    <script defer src="{{ versioned_static('js/modals.js') }}"></script>
 
-{% with %}
-{% set returnURL = "https://canonical.com/solutions/secure-open-source#success" %}
-{% include "solutions/secure-open-source/_form-secure-open-source.html" %}
-{% endwith %}
+    {% with %}
+      {% set returnURL = "https://canonical.com/solutions/secure-open-source#contact-form-success" %}
```
Comment (britneywwc) [inline templates/solutions/secure-open-source/index.html:846]: Change here

**Review by mtruj013 — APPROVED** [review]
👍🏻

**Pull request merged**